src/hotspot/cpu/riscv/vm_version_riscv.cpp

```diff
@@ -182,6 +182,11 @@ void VM_Version::common_initialize() {
   }
 
   if (UseZicboz) {
```

Comment:
I don't think it's safe to turn on `UseBlockZeroing` without knowing the actual cache block size.
So we should add check for that here. Maybe change this if condition into something like:
```
if (UseZicboz && (UseZic64b || zicboz_block_size.enabled())) {
```